src/zoid/checkout/component.jsx

```diff
@@ -85,6 +85,11 @@ export function getCheckoutComponent() : CheckoutComponent {
                     required:   false
                 },
                 
+                stickinessID: {
+                    type:       'string',
+                    queryParam: true
```

Comment:
Can you provide an example value for this new stickinessID prop? Wondering how big they will be and if we need to pass through as a query string param or if we can get this from window.xprops.

Reply:
The ID (ex: `uid_85ecf248f0_mtm6mzy6ntq`) is similar to the sessionID. I believe it is required to be a queryParam since the goal is to have it accessible as part of the checkout state in UL